src/holofuel/components/Header/Header.js

```diff
@@ -8,15 +8,15 @@ import CopyAgentId from 'holofuel/components/CopyAgentId'
 import { presentAgentId } from 'utils'
 
 export function Header ({ title, agent, agentLoading, avatarUrl, history: { push }, hamburgerClick = () => push('/dashboard') }) {
-  const leftNav = <Button onClick={hamburgerClick} styleName='menu-button' dataTestId='menu-button'>
+  const leftNav = <Button onClick={hamburgerClick} styleName='menu-button'>
     <MenuIcon styleName='menu-icon' color='#FFF' />
   </Button>
 
   if (agentLoading) agentLoading = <h4>Loading...</h4>
 
   return <header>
     <section styleName='header'>
-      <div styleName='left-nav'>
+      <div styleName='left-nav' data-testid='menu-button'>
```

Comment:
why move this?

Reply:
Jest wasn't able to find it at the parent the testid at the parent level.